src/reducers/HelpDropdownState.ts

```diff
@@ -5,18 +5,20 @@ import { HelpDropdownActions } from '../actions/HelpDropdownActions';
 
 export const INITIAL_STATUS_STATE: StatusState = {
   status: {},
-  components: [],
-  warningMessages: []
+  externalServices: [],
+  warningMessages: [],
+  istioEnvironment: {
+    isMaistra: false
+  }
 };
 
-// This Reducer allows changes to the 'graphDataState' portion of Redux Store
 const HelpDropdownState = (state: StatusState = INITIAL_STATUS_STATE, action: KialiAppAction): StatusState => {
   switch (action.type) {
     case getType(HelpDropdownActions.statusRefresh):
       return {
         ...INITIAL_STATUS_STATE,
         status: action.payload.status,
-        components: action.payload.components,
+        externalServices: action.payload.components,
```

Comment:
Minor, perhaps worth to adjust the "payload" variable as well (just for clarity).
Action/Reducers are using same variable in the payload, but renaming it.

Again, non blocker, but as actions/reducers in Redux are not trival to read, I think those would help others as well.

![image](https://user-images.githubusercontent.com/1662329/132198662-f29a4c79-bad8-4367-8c26-756cdc98c13a.png)

Reply:
I thought about changing the property names/variable names also to match, but I thought the amount of code changes/churn would be too much. But I understand this would make it more clear if the names all matched. Let me try to do this and see if I can get the names all consistent.